tests/test_mark_tokens.py

```diff
@@ -271,11 +274,12 @@ def test_adjacent_strings(self):
 
 
   def test_print_function(self):
+    # TODO: is this test still needed?
     # This testcase imports print as function (using from __future__). Check that we can parse it.
     # verify_all_nodes doesn't work on Python 2 because the print() call parsed in isolation
     # is viewed as a Print node since it doesn't see the future import
```

Comment:
```suggestion
    # This testcase imports print as function (using from __future__). Check that we can parse it.
```

Reply:
yeah but why do we need to import print from future?

Reply:
This probably isn't testing anything interesting any more but we might as well keep it.